### PR TITLE
visions html and css

### PR DESCRIPTION
**Pull request by alejandromartineza**

### CODE REVIEW

File: pages/vision.html [hunk 1]
```diff
@@ -5,6 +5,36 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
 		<title>Our Vision</title>
+		<link rel="stylesheet" href="../styles/vision_style.css">
 	</head>
-	<body></body>
+	<body>
+		<header>
```
Comment (cloggylabs) [inline pages/vision.html]: Change this header tag for my header tag in index.html file

File: pages/vision.html [hunk 2]
```diff
@@ -5,6 +5,36 @@
...
+		</header>
+		<main>
+			<div class="centro"><img src="https://concepto.de/wp-content/uploads/2016/03/vision.jpg" alt="vision de brujula" class="circulo"><img src="https://media.tenor.com/uXtFY2zyIMsAAAAM/groundhog-day.gif" alt="vision de brujula" class="circulo">
+			</div>
+			<h1 class="centro">vision</h1>
+			<p><img src="https://adm.com.uy/wp-content/uploads/2019/02/destacada-cc-19-02-08.jpg" alt="vision" class="circulo" align="right"><img src="https://66.media.tumblr.com/9dd191e6d5b1eb949a3cff4c057f1c7e/tumblr_olixtzpojT1tlb56zo1_400.gif" alt="poder" class="circulo" align="left"> nuestra vision esta enfocada en el fin del invierno y el inicio del mañana con la primavera dando nuestros aportes creativos fomentando el cambio constante y la evolucion de nuestra imaginacion planteando un pensamiento creando stonks empresariales
+			</p>
+			<div class="centro"><img src="https://media.tenor.com/5BtJ7DfZXIEAAAAM/marmot-isma.gif" alt="poder" class="circulo"></div>
+		</main>
+		<footer>
```
Comment (cloggylabs) [inline pages/vision.html]: same here please

File: styles/vision_style.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.circulo {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+}
+
+.centro {
+    display: flex;
+    justify-content: center;
+}
```
Comment (cloggylabs) [inline styles/vision_style.css]: add the header and footer style in the home_style.css file

**Review by cloggylabs — CHANGES_REQUESTED** [review]
Please check the comments and solve them

File: pages/vision.html (hunk 1 above)
Comment (alejandromartineza) [reply pages/vision.html]: done

File: styles/vision_style.css (hunk 3 above)
Comment (alejandromartineza) [reply styles/vision_style.css]: done

**Review by cloggylabs — APPROVED** [review]

**Pull request merged**